src/components/ReplyField/__tests__/ReplyField-test.tsx

```diff
@@ -40,8 +40,8 @@ describe('ReplyField', () => {
         setCursorPosition: jest.fn(),
     };
 
-    const getWrapper = (props = {}): ShallowWrapper<Props, State, ReplyField> =>
-        shallow(<ReplyField {...defaults} {...props} />);
+    const getWrapper = (props = {}): ReactWrapper<Props, State, ReplyField> =>
+        mount(<ReplyField {...defaults} {...props} />);
```

Comment:
Do we need to mount this component to test the event handler? I think shallow is typically faster, so we should stick with it if possible.